osu.Game/Screens/Multi/Match/RoomSubScreen.cs

```diff
@@ -37,6 +41,37 @@ protected override void LoadComplete()
 
             managerUpdated = beatmapManager.ItemUpdated.GetBoundCopy();
             managerUpdated.BindValueChanged(beatmapUpdated);
+
+            if (music != null)
+                music.TrackChanged += applyToTrack;
```

Comment:
why are you doing this differently to the original code? why not just keep things simple and copy that code across?

by doing this you've called an event leak. there is no unbind of this event binding.

Reply:
Ah, I forgot to unbind..., I intended to rewrite this such that I don't end up with extra useless code (because I changed the place the code was in), but looks like that's a bad practice, will copy exactly how the code was in `Multiplayer`, my bad.

Reply:
I believe I've resolved this. A double check would be appreciated.